visual-inspection/inspection-frontend/src/components/InspectImage.vue

```diff
@@ -51,17 +56,25 @@ export default {
   data() {
     return {
       prediction: null,
+      probability: null,
       backendUrl: process.env.VUE_APP_BACKEND_URL,
-      cancelTokens: []
+      cancelTokens: [],
+      show_probability_percentage: process.env.VUE_APP_SHOW_PROBABILITY_PERCENTAGE
     }
   }
 }
 </script>
 
 <i18n>
 {
-  "de": { "answer": "„Das ist {object}“"},
-  "en": { "answer": "\"This is {object}\""}
+  "de": {
+     "answer1": "„Das ist zu {percentage} % {object}“",
+     "answer2": "„Das ist {object}“"
```

Comment:
```suggestion
     "answer_with_probability": "„Das ist zu {percentage} % {object}“",
     "answer_without_probability": "„Das ist {object}“"
```

Wir sollten hier sprechendere Namen vergeben (muss dann natürlich im Rest auch angepasst werden)